lib/salus/scanners/report_pom_xml.rb

```diff
@@ -0,0 +1,38 @@
+require 'salus/scanners/base'
+
+# Report Java library usage
+
+module Salus::Scanners
+  class ReportPomXml < Base
+    def run
+      shell_return = run_shell("bin/parse_pom_xml #{@repository.path_to_repo}/pom.xml", chdir: nil)
+
+      if !shell_return.success?
+        report_error(shell_return.stderr)
+        return
+      end
+
+      dependencies = JSON.parse(shell_return.stdout)
+
+      dependencies.each do |dependency|
+        group_id = dependency['group_id']
+        artifact_id = dependency['artifact_id']
+        version = dependency['version']
+        report_dependency(
+          'pom.xml',
+          type: 'maven',
+          name: "#{group_id}.#{artifact_id}",
+          version: version.nil? ? 'unknown' : version
```

Comment:
I do not know if `unknown` is a valid version value in cyclonedx.  
Could you find a cyclondex schema or something to check?  If it's an invalid value, then when our service processes the the cyclondex, then everything could have blank values (happened with our license scanner).